src/analysis/Analyses.ts

```diff
@@ -520,12 +514,12 @@ export const BaseAnalyses = (
         .Workspaces.workspace(workspaceInfo.namespace, workspaceInfo.name)
         .listActiveFileTransfers();
       setActiveFileTransfers(!_.isEmpty(fileTransfers));
-
-      await refreshRuntimes();
```

Comment:
This is already happening both when the `WorkspaceContainer` is created, and then on a polling interval, also in `WorkspaceContainer`